.gitignore

```diff
@@ -36,4 +36,3 @@ test-*.dart
 **/coverage/**
 coverage.json
 lcov.info
-.vscode/
```

Comment:
Don't see why we need to remove this. We don't have any nyxx-specific config, but developers (me included) store their launch configurations in here which shouldn't be committed.

Reply:
@abitofevrything Sorry, late reply, but .vscode is already ignored on top of the file